src/components/Layout.res

```diff
@@ -195,21 +159,7 @@ let make = () => {
                   ReactEvent.Mouse.preventDefault(event)
                   clearAndPush(j`/#leaderboards/monthly-contribution`)
                 }}>
-                {"LEADERBOARDS"->restr}
-              </a>,
-          },
-          {
-            shouldDisplay: true,
-            shouldDisplayMobile: true,
-            component: (closeModal, _) =>
-              <a
-                className=Styles.navListText
-                onClick={event => {
-                  closeModal()
-                  ReactEvent.Mouse.preventDefault(event)
-                  clearAndPush(j`/#dao`)
-                }}>
-                {"DAO"->restr}
+                {"TOP GUARDIANS"->restr}
```

Comment:
The term leaderboards apparently wasn't super clear (unless you've been on wildcards before). Maybe this isn't that much better, but at least it prompts the user along as to what it means I think.

Reply:
This looks great! Much better than leaderboards. More conservation facing